docs/articles/documentation/test-api/authentication/user-roles.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: docs
+title: User Roles
+permalink: /documentation/test-api/authentication/user-roles.html
+---
+# User Roles
+
+Many test scenarios involve the activity of more than one user. TestCafe addresses these scenarios by providing a convenient way
+to isolate authentication test actions and apply them easily whenever you need to switch the user account.
+
+A piece of logic that logs in a particular user is called a *role*. Define a role for each user account participating in your test.
+
+## Creating and Using Roles
+
+Use the `Role` constructor to create and initialize a role.
+
+```text
+Role( func( t ) )
```

Comment:
Oh, I didn't mention an elephant in the room! Role ctor always accepts two argument, first being a login page URL.